tests/scripts/check_alert_tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
if the scripts are only for Prom, can move into `tests/prometheus/script`s?

Reply:
Sure, I can move it to this dir level